vendor/engines/secure_rooms/app/controllers/secure_rooms/facility_occupancies_controller.rb

```diff
@@ -25,6 +25,10 @@ def initialize
       @active_tab = "admin_occupancies"
     end
 
+    def dashboard
+      @secure_rooms = current_facility.products(SecureRoom).active
```

Comment:
We default all of our secure rooms to `is_hidden`. I think we need `active_plus_hidden`.